tests/pipeline/test_dividends.py

```diff
@@ -184,19 +184,19 @@ def get_dataset(cls):
     def setup(self, dates):
         date_intervals = [
             [
-                [None, '2014-01-04'], ['2014-01-05', '2014-01-09'],
-                ['2014-01-10', None]
+                ['2014-01-01', '2014-01-04'], ['2014-01-05', '2014-01-09'],
+                ['2014-01-10', '2014-01-31']
             ],
             [
-                [None, '2014-01-04'], ['2014-01-05', '2014-01-09'],
-                ['2014-01-10', None]
+                ['2014-01-01', '2014-01-04'], ['2014-01-05', '2014-01-09'],
+                ['2014-01-10', '2014-01-31']
             ],
             [
-                [None, '2014-01-04'], ['2014-01-05', '2014-01-14'],
-                ['2014-01-15', None]
+                ['2014-01-01', '2014-01-04'], ['2014-01-05', '2014-01-14'],
+                ['2014-01-15', '2014-01-31']
             ],
             [
-                [None, '2014-01-04'], ['2014-01-05', None]
+                ['2014-01-01', '2014-01-04'], ['2014-01-05', '2014-01-31']
```

Comment:
I just wanted to have some specific begin/end date to search for in the date index. I could also put the `None`'s back and change the search logic to assume the first/last element of the date index.